ingress/nginx.tmpl

```diff
@@ -0,0 +1,44 @@
+worker_processes  1;
+daemon off;
+
+error_log  /dev/stdout warn;
+pid        {{ .WorkingDir }}/nginx.pid;
+
+events {
+ worker_connections  1024;
+}
+
+http {
+ default_type  application/octet-stream;
+
+ log_format  main  '$remote_addr - $remote_user [$time_local] "$request" '
+ '$status $body_bytes_sent "$http_referer" '
+ '"$http_user_agent" "$http_x_forwarded_for"';
+
+ access_log  /var/log/nginx/access.log  main;
+ sendfile        on;
+ keepalive_timeout  65;
+
+ {{ $port := .Port }}
+ {{ range $entry := .Entries }}
+ # Start entry
+ server {
+   listen {{ $port }};
+   server_name {{ $entry.Host }};
+   location {{ $entry.Path }} {
+     proxy_pass http://{{ $entry.ServiceName }}:{{ $entry.ServicePort }};
```

Comment:
We should pass in the client ip the ELBs provide us - this will be required by app teams I think, and will be important for debugging issues when external things start connecting. Can get this from the x-forward-for header from the ELB I believe.